app/views/users/show.html.erb

```diff
@@ -21,21 +21,23 @@
         </div>
       </div>
       <div class="table-responsive">
-        <table class="table table-bordered table-striped table-hover">
+        <table id="Task-table" class="table table-bordered table-striped table-hover">
           <thead>
             <tr>
               <th>Task Name</th>
               <th>Description</th>
               <th>Date</th>
+              <th>  </th>
             </tr>
           </thead>
 
           <tbody>
             <% @items.each do |item| %>
-              <tr>
+              <tr id=<%= "task-#{item.id}" %>">
```

Comment:
```html
<tr id=<%= "task-#{item.id}" %>">
```
The opening double-quote for the id attribute is missing, try:
```html
<tr id="task-<%= item.id %>">
```
That might fix the JS problem mentioned.

It'd be ok to simplify the selector in the javascript file to just use the id (drop the `tr`): 
```html
$('#task-' + <%= @item.id %>).hide();
```